src/tests/Microsoft.PowerFx.Repl.Tests.Shared/ModuleTests.cs

```diff
@@ -0,0 +1,293 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.VisualBasic.Syntax;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+#pragma warning disable CS0618 // Type or member is obsolete
+#pragma warning disable SA1118 // Parameter should not span multiple lines
+
+namespace Microsoft.PowerFx.Repl.Tests
+{
+    // Thesde tests should target public Module Import() function 
+    // (and not internal services)
+    public class ModuleTests
+    {
+        private PowerFxREPL _repl;
+        private readonly TestReplOutput _output = new TestReplOutput();
+
+        public ModuleTests()
+        {
+            var config = new PowerFxConfig();
+            config.SymbolTable.EnableMutationFunctions();
+
+            // config.EnableSetFunction();
+            var engine = new RecalcEngine(config);
+
+            _repl = new PowerFxREPL
+            {
+                Engine = engine,
+                Output = _output,
+                AllowSetDefinitions = true,
+                AllowUserDefinedFunctions = true,
+                AllowImport = true,
+                ParserOptions = new ParserOptions() { AllowsSideEffects = true }
+            };
+        }
+        
+        // Run line, return the normal output. 
+        private string HandleLine(string fx, bool expectErrors = false)
+        {
+            Debugger.Log(0, string.Empty, ">> " + fx);
+            _repl.HandleLine(fx);
+
+            if (!expectErrors)
+            {
+                AssertNoErrors();
+            }
+
+            var log = _output.Get(OutputKind.Repl);
+
+            Debugger.Log(0, string.Empty, log);
+            return log;
+        }
+
+        private void AssertNoErrors()
+        {
+            Assert.Empty(_output.Get(OutputKind.Error));
+        }
+
+        private string GetModuleFullPath(string path)
+        {
+            string fullpath = Path.Combine(
+                Environment.CurrentDirectory,
+                "Modules",
+                path);
+
+            return fullpath;
+        }
+
+        private void Import(string path, bool expectErrors = false)
+        {
+            string fullpath = GetModuleFullPath(path);
+
+            var expr = $"Import(\"{fullpath}\")";
```

Comment:
>Import [](http://example.com/codeflow?start=25&length=6)

Do we want Import as a function or just as a structural piece of the surrounding yaml?

Reply:
Definitely *not* a function (yet). Then that raises concerns like conditioan imports (nested in an If) or computing the path dynamically. Safer to restrict to just in the yaml.